src/test/store/receive-profile.test.js

```diff
@@ -585,19 +596,47 @@ describe('actions/receive-profile', function () {
     });
 
     it('should be able to switch to full view from the active tab', function () {
-      const { dispatch, getState } = setup(true);
+      const { dispatch, getState } = setup({ initializeCtxId: true });
+      expect(
+        UrlStateSelectors.getTimelineTrackOrganization(getState())
+      ).toEqual({
+        type: 'active-tab',
+        tabID,
+      });
+      dispatch(changeTimelineTrackOrganization({ type: 'full' }));
+      expect(
+        UrlStateSelectors.getTimelineTrackOrganization(getState())
+      ).toEqual({
+        type: 'full',
+      });
+    });
+
+    it('should reset the url state while switching to the full view', function () {
```

Comment:
This new test was failing before and passing now.

Reply:
Thanks for the new test!

Reply:
I wonder if the previous test is still useful, the new test seems to include it completely?

Reply:
Yeah, it's not super useful anymore but I think it's still nice to have a simpler test that fails before this, so it's better reproducible and have a better test name for context.